packages/autorest.typescript/src/autorestSession.ts

```diff
@@ -44,8 +44,6 @@ export interface AutorestOptions {
   lenientModelDeduplication?: boolean;
   useLegacyLro?: boolean;
   flavor?: PackageFlavor;
-  //TODO should remove this after finish the release tool test
-  shouldUsePnpmDep?: boolean;
```

Comment:
i remember automation has used this option, @skywing918 do we need any update there?

Reply:
`shouldUsePnpmDep` is not used in automation.